### ISSUE TITLE
Add WANTED Adventures to GetProfile.CharacterActivities[204]

### ISSUE THREAD

**Issue by lowlines** [Bungie-net/api#711]
Every week, Spider sells an orange WANTED bounty to go kill a boss enemy that is hiding in one or more Lost Sectors on a given destination. For some locations, the Lost Sector actually rotates on a daily basis allowing players to choose whether they complete the bounty in a more challenging location. Would it be possible to return the daily activityHash?

---

EDZ Bounty:
1704959157 WANTED: Silent Fang

EDZ Activities:
494635832 WANTED: Silent Fang (Winding Cove)
1111101131 WANTED: Silent Fang
3679946187 WANTED: Silent Fang

Titan Bounty:
2277930478 WANTED: Gravetide Summoner

Titan Activities:
186006588 WANTED: Gravetide Summoner
386959931 WANTED: Gravetide Summoner (DS Quarters-2)
497583046 WANTED: Gravetide Summoner (Methane Flush)

Nessus Bounty:
1781061075 WANTED: Arcadian Chord

Nessus Activities:
2431838030 WANTED: Arcadian Chord
3676143304 WANTED: Arcadian Chord
3774573332 WANTED: Arcadian Chord

Io Bounty:
2755895280 WANTED: Blood Cleaver

Io Activities
1003955024 WANTED: Blood Cleaver (Grove of Ulan-tan)
3957909528 WANTED: Blood Cleaver (Sanctum of Bones)
4275462311 WANTED: Blood Cleaver (Aphix Conduit)

Mercury Bounty:
3805779101 WANTED: Combustor Valus

Mercury Activity:
1835551337 WANTED: Combustor Valus

Mars Bounty:
3321772950 WANTED: Eye in the Dark

Mars Activities:
4150577752 WANTED: The Eye in the Dark
4199058482 WANTED: The Eye in the Dark

---

There should also be a locationHash for each of these activities as I've been checking against the data returned of the APIs and have seen them.

**Comment by ttgmpsn** [comment]
Adventures unfortunately are no CharacterActivities for the API - you can't get the current heroic adventure for the flashpoint that way, either.

**Comment by lowlines** [comment]
That's not true. Adventures are an activity much like patrol missions and even public events, the latter which being the only type of activity in the game that doesn't have an ActivityDefinition in the manifest. That being said, both public events and adventures are marked in the Director, so it should be possible to return this state to get the activityHash for the CharacterActivities component.

**Comment by ttgmpsn** [comment]
Sorry, I was referring to what that endpoint (CharacterActivities) returns, which is (as of now) just stuff that you can launch into directly from the director. There are many nodes & missions on the worldmap that you can select and track, but that aren't returned by CharacterActivities.

(Not that I don't want this to change or see CharacterActivities expanded, see #432)

**Comment by vthornheart-bng** [comment]
TFS 717538
